services/121-service/src/fsp/dto/update-fsp.dto.ts

```diff
@@ -61,7 +61,7 @@ export class CreateFspAttributeDto extends UpdateFspAttributeDto {
 
 export class UpdateFspDto {
   @ApiProperty({
-    example: { en: 'FSP PA-app display name', nl: 'FSP PA-app weergavenaam' },
+    example: { en: 'FSP display name', nl: 'FSP weergavenaam' },
```

Comment:
I'm not confident about this change but this seemed to be the case from some (shallow) searching.

Reply:
I thin changes are fine in this line, the purpose of this text is just to show it in swagger documentation.

Reply:
Yeah, that part is clear to me. What I'm not sure about is whether it is true that this is now simply display name, or whether it is actually unused in the portal (and therefore should be removed).

Reply:
Its fine, As far I know, its simply `FSP display name` which is configurable by program now and display it accordingly in the portal.